rlp/sedes/big_endian_int.py

```diff
@@ -18,7 +17,7 @@ def __init__(self, l=None):
         self.l = l
 
     def serialize(self, obj):
-        if not is_integer(obj):
+        if not isinstance(obj, int):
             raise SerializationError('Can only serialize integers', obj)
```

Comment:
Might as well explicitly document that this can (now) also encode `bool`

Reply:
I'm not sure that's something we actually want.  I'm going to add a check to forbid bool.